llvm-remote-index.md

```diff
@@ -0,0 +1,79 @@
+# LLVM remote index service
+
+We plan to provide [remote index](/remote-index.md) service for LLVM
+developers, this document describes the details of this service.
+
+We provide our best effort support for remote index service but it is not an
+official product by either LLVM or Google.
+
+**NOTE**: The service is currently in the alpha testing stage and we would
+like to gradually expand the scope of availability and feature set.
+
+## Overview
+
+LLVM remote index service is implemented by the team maintaining clangd. We
+plan to provide the service to LLVM developers to lower the entry bar for
+developers with less powerful machines and make the development process more
+accessible. The whole service implementation, including infrastructure, is
+open source and publicly available
+([clangd/index/remote](https://github.com/llvm/llvm-project/tree/master/clang-tools-extra/clangd/index/remote)
+for `clangd-index-server` and client implementation,
+[llvm-remote-index](https://github.com/clangd/llvm-remote-index) for
+indexing, data pipelines and deployment). Google donated Google Cloud
```

Comment:
Maybe change the last sentence to:
`The service is hosted on GCP by clangd team, with sponsorship of Google`

Does that SG @ChristianKuehnel ?

Reply:
SG. To be honest, I'm not sure if wording makes a difference in the end.